hack/setup_db.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
```

Comment:
`setup` sounds a bit like installation, rename file to `start_db.sh`

Reply:
sure, will do.